tests/microos/enable_nested_virt.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Enable nested virtualization support
```

Comment:
Isn't this something you need to enable on the outer host (i.e. openQA worker) only?

The other virtualization tests (and openQA-in-openQA) don't seem to enable that either.

Reply:
It works out of the box, I can drop this module. Thanks
http://kepler.suse.cz/tests/6153#